### PR TITLE
Update var to let/const

### PR DESCRIPTION
**Pull request by adeobas**
#### Summary
Made changes to: 

Page: Int16Array

#### Motivation
Line 107: var to const 
Line 114: var to const 
Line 118: var to let 
Line 119: var to const 
Line 123: var to let 
Line 124: var to const 
Line 127: var to let 
Line 128: var to const

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/int16array/index.md [hunk 1]
```diff
@@ -104,28 +104,28 @@ The **`Int16Array`** typed array represents an array of twos-complement 16-bit s
...
-var x = new Int16Array([21, 31]);
-var y = new Int16Array(x);
+let x = new Int16Array([21, 31]);
+const y = new Int16Array(x);
 console.log(y[0]); // 21
 
 // From an ArrayBuffer
-var buffer = new ArrayBuffer(8);
-var z = new Int16Array(buffer, 0, 4);
+let buffer = new ArrayBuffer(8);
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/int16array/index.md:123]:
```suggestion
const buffer = new ArrayBuffer(8);
```

File: files/en-us/web/javascript/reference/global_objects/int16array/index.md [hunk 2]
```diff
@@ -104,28 +104,28 @@ The **`Int16Array`** typed array represents an array of twos-complement 16-bit s
...
 
 // From an array
-var arr = new Int16Array([21,31]);
+const arr = new Int16Array([21,31]);
 console.log(arr[1]); // 31
 
 // From another TypedArray
-var x = new Int16Array([21, 31]);
-var y = new Int16Array(x);
+let x = new Int16Array([21, 31]);
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/int16array/index.md:118]:
```suggestion
const x = new Int16Array([21, 31]);
```

File: files/en-us/web/javascript/reference/global_objects/int16array/index.md [hunk 3]
```diff
@@ -104,28 +104,28 @@ The **`Int16Array`** typed array represents an array of twos-complement 16-bit s
...
-var buffer = new ArrayBuffer(8);
-var z = new Int16Array(buffer, 0, 4);
+let buffer = new ArrayBuffer(8);
+const z = new Int16Array(buffer, 0, 4);
 
 // From an iterable
-var iterable = function*(){ yield* [1,2,3]; }();
-var int16 = new Int16Array(iterable);
+let iterable = function*(){ yield* [1,2,3]; }();
+const int16 = new Int16Array(iterable);
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/int16array/index.md:128]:
```suggestion
const int16_from_iterable = new Int16Array(iterable);
```

**Review by wbamberg — DISMISSED** [review]
Thank you @adeobas !

I think those `let` declarations can be `const`, because they are not being reassigned (set to point at something else).

Also we need to use a different name for the second `int16` variable, because we are not allowed to redeclare variables that were declared using `const` (this is allowed with `var`).

File: files/en-us/web/javascript/reference/global_objects/int16array/index.md [hunk 4]
```diff
@@ -104,28 +104,28 @@ The **`Int16Array`** typed array represents an array of twos-complement 16-bit s
...
 // From an ArrayBuffer
-var buffer = new ArrayBuffer(8);
-var z = new Int16Array(buffer, 0, 4);
+let buffer = new ArrayBuffer(8);
+const z = new Int16Array(buffer, 0, 4);
 
 // From an iterable
-var iterable = function*(){ yield* [1,2,3]; }();
-var int16 = new Int16Array(iterable);
+let iterable = function*(){ yield* [1,2,3]; }();
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/int16array/index.md:127]:
```suggestion
const iterable = function*(){ yield* [1,2,3]; }();
```

**Review by teoli2003 — APPROVED** [review]
Thanks all!

**Pull request merged**